crates/editor/src/mouse_context_menu.rs

```diff
@@ -169,6 +177,9 @@ pub fn deploy_context_menu(
                 .separator()
                 .action("Rename Symbol", Box::new(Rename))
                 .action("Format Buffer", Box::new(Format))
+                .when(!selections, |builder| {
```

Comment:
Should the button be shown only when a LSP supports range formatting or something? Maybe something like a flag in settings a-la `formatSelectionEnabled: true`?